src/panel/app_panel.js

```diff
@@ -18,7 +18,7 @@ import DirectionPanel from './direction/direction_panel';
 import Menu from './menu';
 import Telemetry from '../libs/telemetry';
 import CategoryPanel from './category_panel';
-import PanelManager from 'src/proxies/panel_manager';
```

Comment:
so "panel_manager.js" file can be safely deleted, no ?

Reply:
Ah yes, just forgot. Thanks!